anfema_django_testutils/templates/test-results-template.html

```diff
@@ -61,136 +63,121 @@ <h1>{{ title }}</h1>
                 </table>
             </div>
             <div class="container">
-                {% for testcase, testcase_results in testcases.items %}
-                    <div class="testcase">
-                        <table>
-                            <thead>
-                                <tr>
-                                    <th>{{ testcase }}</th>
-                                    <th>Duration</th>
-                                    <th>
-                                        Result
-                                    </th>
-                                    <th>
-                                        <button class="btn-testcase-details">
-                                            &ctdot;
-                                        </button>
-                                    </th>
-                                </tr>
-                                <tr>
+                <form action="/action_page.php">
```

Comment:
we can drop the action probably :smile:

Reply:
Insight gained: excessive copy & paste leads to errors.